src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitedDate = date.split(fromFormat[3]);
+
+  const dateIndexFrom = fromFormat.indexOf('DD');
```

Comment:
I guess this variable should named as `dayIndexFrom` and `dayIndexTo`

Reply:
In my opinion day is like weekday and date is like month-date. That is why I chose date

Reply:
I have revised my solution using Object, please re-review